crowdstrikeoauthapi_connector.py

```diff
@@ -1668,6 +1681,74 @@ def _handle_list_detections(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_get_detections_details(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
```

Comment:
```suggestion
        self.save_progress(f"In action handler for: {self.get_action_identifier()}")
```

Reply:
As per our developer's best practice, we are adding `format string` or `f-string` for string operations.

Reply:
Do you mean this point 

> Consistent text formatting for all labels and messages.

from here https://splunk.atlassian.net/wiki/spaces/APD/pages/253727933350/Code+Review+Guidelines? Or some other document

Reply:
We have our developer best practice sheet that we are following currently.